src/components/MarketDetail/index.js

```diff
@@ -195,8 +206,8 @@ class MarketDetail extends Component {
       <div className="marketControls container">
         <div className="row">
           {Object.keys(expandableViews).filter(view =>
-            !expandableViews[view].showCondition ||
-            expandableViews[view].showCondition(this, market),
+            typeof expandableViews[view].showCondition !== 'function' ||
+            expandableViews[view].showCondition(this.props, market),
```

Comment:
If this is also used at the other place where we check for the showCondition, remove it there too please